BoxContentSDK/BoxContentSDK/OAuth2/BOXAuthorizationViewController.m

```diff
@@ -102,7 +104,7 @@ - (instancetype)initWithSDKClient:(BOXContentClient *)SDKClient
 
         NSURLSessionConfiguration *sessionConfiguration = [NSURLSessionConfiguration defaultSessionConfiguration];
         _URLSession = [NSURLSession sessionWithConfiguration:sessionConfiguration
-                                                    delegate:self
+                                                    delegate:self //NOTE: The URL Session holds a strong reference on its delegate until it is invalidated
```

Comment:
Per Apple's docs, this this is a 'strong' reference, which does not get cleaned up until the delegate is set to something else or the session is invalidated.